partials/restaurant-info.html

```diff
@@ -2,8 +2,8 @@
   <header>
     <h2>{{name}}</h2>
   </header>
-  <div class="restaurant-website"><a href="{{website}}">Website</a> | <a href="{{reservations}}">Reservations</a></div>
+  <div class="restaurant-website"><a href="{{formatWebsite website}}" target="_blank">Website</a></div>
```

Comment:
This change eliminates the link to reservations, which is totally fine, if that's what you intended. If we do want it to show up, we'd need to add a custom field to Trello for it.